src/modules/fw_autotune_attitude_control/fw_autotune_attitude_control.cpp

```diff
@@ -485,8 +485,7 @@ void FwAutotuneAttitudeControl::updateStateMachine(hrt_abstime now)
 	// the identification sequence is aborted immediately
 	if (_state != state::wait_for_disarm && _state != state::idle && _state != state::fail && _state != state::complete) {
 		if (now - _state_start_time > 20_s
-		    || (_param_fw_at_man_aux.get() && !_aux_switch_en)
-		    || _start_flight_mode != _nav_state) {
+		   ) {
```

Comment:
why?

Reply:
Sorry I try someting at SITL and I forget it I will check and correct it